server/leafnode.go

```diff
@@ -1218,7 +1229,15 @@ func (c *client) processLeafNodeConnect(s *Server, arg []byte, lang string) erro
 	c.headers = supportHeaders && proto.Headers
 
 	// Remember the remote server.
-	c.leaf.remoteServer = proto.Name
+	// We changed the leafnode CONNECT.Name json tag from "name" to "server_name",
```

Comment:
Note: I could remove this comment and code (just have c.leaf.remoteServer = proto.Name) if I revert the change from "name" to "server_name" that you did in JSC.

Reply:
ok sounds good.

Reply:
Just double checking: sounds good mean that you agree on changing back "server_name" json tag to "name"?